src/test/kotlin/heist/make/sdk/kt/sse/EventSourceTest.kt

```diff
@@ -0,0 +1,35 @@
+package heist.make.sdk.kt.sse
+
+import heist.make.sdk.kt.event.Event
+import org.junit.jupiter.api.Assertions
+import kotlin.test.Test
+
+class EventSourceTest {
+
+    private fun handleMessage( e: Event ) {
+
+        System.out.println( ( e as MessageEvent ).data )
+    }
+
+    private fun handleMessage2( e: Event ) {
+        Assertions.assertEquals(
```

Comment:
We should use Jackson or smth to serialize the result into JSON and check if its correct.